beetmoverscript/docker.d/worker.yml

```diff
@@ -17,6 +17,7 @@ bucket_config:
             key: { "$eval": "DEP_KEY" }
           buckets:
             devedition: 'net-mozaws-stage-delivery-archive'
+            pinebuild: 'net-mozaws-stage-delivery-archive'
```

Comment:
all the other values are aligned, can you add an extra whitespace after `pinebuild:`